### DISCUSSION TITLE
How to turn off italics for jsx props?

### DISCUSSION THREAD

**Discussion by skoch13** [catppuccin/nvim#595] [Q&A] [answered]
The only solution I found is to disable italics entirely, but I can hardly consider this as a satisfactory solution.

**Comment by diegoulloao** [comment]
> The only solution I found is to disable italics entirely, but I can hardly consider this as a satisfactory solution.

were you able to find a solution?

**Reply by skoch13** [reply]
Nope. I switched to another theme 🧐

**Comment by skoch13** [comment, marked as answer]
The solution for `tsx` would be the following:

```
custom_highlights = function(C)
  return {
          ["@tag.attribute.tsx"] = { fg = C.teal, style = {} },
        }
      end,
    },
```
